devins bio (#39)
chloerice/wbb-3.0: content/team.md

  - name: Devin Jackson
    role: CEO and Co-founder
    headshotPath:

>     bio: Devin is a Full Stack Engineer with 5 years experience building products with meaning and love for finding people's hidden potential. His hobbies include debating Hip Hop, riding his bike, and sipping maple syrup whiskey.

Reviewer: ```suggestion
    bio: Devin is a Full Stack Engineer with 5 years of  experience building products with meaning. He loves finding people's hidden potential. His hobbies include debating Hip Hop, riding his bike, and sipping maple whiskey.
```
